remoulade/state/message_state.py

```diff
@@ -0,0 +1,44 @@
+from ..cancel import Cancel
+from .backend import State
+
+
+class Decorator:
+    """Decorator to save the
+    state in a backend
+    """
+    def save(state):
+        def wrapper(decorated):
+            def save_state(*args, **kwargs):
+                key = args[2].message_id
+                args[0].backend.store_result(key,
+                                             State(state),
+                                             args[0].result_ttl)
+            return save_state
+        return wrapper
+
+
+class MessageState(Cancel):
+
+    """Middleware use to storage and update the state
+    of the messages.
+    """
+    def __init__(self, backend, result_ttl=86400):
```

Comment:
`result_ttl` should be `status_ttl` How put the default in the doc, how much is 86400, what unit is that. I think that 24h if a safe default.